build_daemon/lib/client.dart

```diff
@@ -0,0 +1,129 @@
+import 'dart:async';
+import 'dart:convert';
+import 'dart:io';
+
+import 'package:web_socket_channel/io.dart';
+
+import 'constants.dart';
+import 'data/build_options_request.dart';
+import 'data/build_request.dart';
+import 'data/build_status.dart';
+import 'data/build_target_request.dart';
+import 'data/log_to_paths_request.dart';
+import 'data/serializers.dart';
+import 'data/server_log.dart';
+
+class BuildDaemonClient {
+  IOWebSocketChannel _channel;
+
+  final _buildResultsStreamController =
+      StreamController<BuildResults>.broadcast();
+
+  final _serverLogStreamController = StreamController<ServerLog>.broadcast();
+
+  BuildDaemonClient._();
+
+  Stream<BuildResults> get buildResultsStream =>
```

Comment:
Is the `Stream` part of the name adding value?

Reply:
Done.